modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/jaxrs/writer/interceptor/NestedFieldsWriterInterceptor.java

```diff
@@ -73,6 +76,7 @@
 /**
  * @author Ivica Cardic
  */
+@Priority(Priorities.USER - 10)
```

Comment:
Not sure if this annotation will work as we are registering by using the method `register` in `VulcanFeature` class.

If not, we should try in `VulcanFeature.java`:

```
		_nestedFieldsWriterInterceptor = new NestedFieldsWriterInterceptor(
			_bundleContext, Priorities.USER - 10);
```

https://docs.oracle.com/javaee/7/api/javax/ws/rs/core/Configurable.html#register-java.lang.Class-int-

Reply:
True, thx for catching that, guess I was just lucky with non-deterministic behavior :smile: